### DISCUSSION TITLE
UI proposition

### DISCUSSION THREAD

**Discussion by bbbbbgdn** [cables-gl/cables_docs#447] [Feature Requests] [answered]
Overall request for bigger ports in the ops! There's plenty of space in the OP why should we aim into the ports, could we just enlarge it?  

  
![Screenshot 2023-01-03 at 16 34 14](https://user-images.githubusercontent.com/12091461/210391071-b9d856d5-b755-417b-adf7-a62796d11078.png)

**Comment by bbbbbgdn** [comment]
Rough sketch what I mean.

**Comment by bbbbbgdn** [comment]
I guess overall it's something like a step in between of the classic small ports, and going touchscreen-way, with this new hold-drag-connect feature.

**Reply by steam0r** [reply]
you know you can just drag links onto the op itself and then be presented with possible ports, right?

also op-size is basically controlled by how many ports it has, and we like smaller ops - for other reasons as well.

**Comment by bbbbbgdn** [comment, marked as answer]
Is there any way I could mess with it for my self locally?
Just for the sake of custom theme, more like custom 'CSS' with tweaks?

**Reply by steam0r** [reply]
the whole editor in not using html/css, it's all in webgl

**Reply by bbbbbgdn** [reply]
Yeah I understand it. Custom CSS was there just for comparison. 
But the question is, can I make any local changes to the interface, without hosting separate cables instance or something like this?

**Reply by steam0r** [reply]
no, that's currently not possible and most likely will never really be "without hosting a seperate instance" or haveing some kind of local development/custom version
